File: exercises-cpp/3. Repetitivas/14.cpp
#include<iostream>
using namespace std;

int main() {
    system("cls");
    int numero;
    cout<<"Numero: "; cin>>numero;

    int a = 0;
    for(int i=1; i<=numero; i++){
        if( numero % i == 0 ) { a++; }
    }

    if(a==2) {cout<<"Es primo";}
    else {cout<<"No es primo";}

    return 0;
}

File: exercises-python/3. Repetitivos/14.py
'''
14.Un número es primo si tiene únicamente dos divisores: 1 y el mismo. Desarrolle el programa.
'''

import os
os.system("cls")

numero = int(input("Número: "))
divisores = 0

for i in range(1,numero+1):
    if numero % i == 0:
        divisores += 1
if divisores == 2: print("Es primo.")
else: print("No es primo.")